### PR TITLE
docs: Attempt to clarify use of `where i32: ConvertTo<T>

### PR DESCRIPTION
**Pull request by rjbs**
I read this section a few times before even having a guess what
was meant, then consulted IRC for confirmation.  It may be that I
was thick-headed, but I think this is a useful addition.

### CODE REVIEW

File: src/doc/trpl/traits.md [hunk 1]
```diff
@@ -400,7 +400,9 @@ fn inverse<T>() -> T
 
 This shows off the additional feature of `where` clauses: they allow bounds
 where the left-hand side is an arbitrary type (`i32` in this case), not just a
-plain type parameter (like `T`).
+plain type parameter (like `T`).  In this example, `i32` must implement
```
Comment (apasel422) [inline src/doc/trpl/traits.md]: Can you change this to one space after the period? Thanks!

**Comment by rjbs** [comment]
I have replaced two spaces after full stops with single spaces!

**Comment by steveklabnik** [comment]
Thank you!

**Pull request merged**